### ISSUE TITLE
Add theme patching using env parameters

### ISSUE THREAD

**Issue by realies** [thelounge/thelounge-docker#133]
Since thelounge does not support custom theme-name meta name values, could this be optionally added here via an environment variable? For example, if I'm making all #415364 fields black using Docker post arguments like these at the moment:

> bash -c "sed -i 's/415364/000000/' /usr/local/share/.config/yarn/global/node_modules/thelounge/{public/thelounge.webmanifest,client/index.html.tpl} && docker-entrypoint.sh thelounge start"

**Comment by williamboman** [comment]
Hello! Hmm not sure I'm following what you mean by custom theme-name meta name values. Do you mean like parameterizing themes, to allow for user overrides?

Is there a broad use case/need for this? Otherwise you could either override the container's `entrypoint`, or extend the image with your own Dockerfile, with your custom entrypoint:

```sh
#!/bin/bash

sed -i 's/415364/000000/' /usr/local/share/.config/yarn/global/node_modules/thelounge/{public/thelounge.webmanifest,client/index.html.tpl}

# The original docker-entrypoint.sh exists on PATH
docker-entrypoint.sh "$@"
```

**Comment by realies** [comment]
Yes, I mean customising theme parameters such as the meta theme-name natively by this Docker image. I know that this is currently not possible via thelounge itself, and probably it will be a while until it is. Do you think it's appropriate to have this functionality in here?

**Comment by brunnre8** [comment]
not a packaging bug
